### DISCUSSION TITLE
Security issue in Ghidra

### DISCUSSION THREAD

**Discussion by wbierman** [NationalSecurityAgency/ghidra#6888] [Q&A] [answered]
Greetings!  I have discovered a bug which may be a security issue in Ghidra. I have an explanation and a suggested fix. I'm not sure of the security implications. Is there a way to submit this privately, just in case?

**Comment by ryanmkurtz** [comment, marked as answer]
Hello, you can email me at `ryanmkurtz.git@gmail.com` if that works for you.
